orchestra/models/communication/mixins.py

```diff
@@ -41,6 +41,18 @@ def get_request_cause_description(self):
     def get_request_status_description(self):
         return self.Status(self.status).description
 
+    def is_open(self):
+        if self.status == self.Status.PROCESSING.value:
+            return True
+        inquiries = self.inquiries.all()
+        num_responses = 0
+        for inquiry in inquiries:
```

Comment:
rather than looping through all inquiries which involves a lot of round trips to the DB, can you write two queries?
* q1: are there any responses for this request that are winners?
* q2: count the number of responses per inquiry, and count how many inquiries have no responses.